plaidml/bridge/openvino/ops/reorg_yolo.cpp

```diff
@@ -21,7 +21,7 @@ void registerReorgYolo() {
     auto I = ctx.operands.at(0);
     // as openvino op request, the strides have to be divisible by 'H' and 'W'.
     auto strides = layer->get_strides()[0];
-    return edsl::make_tuple(op::reorg_yolo(I, strides, false));
+    return edsl::make_tuple(op::reorg_yolo(I, strides, true));
```

Comment:
There was no reference implementation for reorg_yolo in nGraph. They added reorg_yolo reference implementation recently (included in this update). 

Unfortunately, this reference implementation does not have the `decrease` flag as plaidml op::reorg_yolo does and it does not match `op::reorg_yolo(I, strides, false)` or  `op::reorg_yolo(I, strides, true);` We though there was 3 different ways to fix this:
1. change the nGraph reference implementation to match `op::reorg_yolo(I, strides, true)`
2. change the nGraph reference implementation to match `op::reorg_yolo(I, strides, false)`
3. change plaidml `op::reorg_yolo `to match  nGraph reference implementation

I can not say which implementation is right or wrong because I think there were arguments about some details how this op should be implemented. But I believe plaidml `op::reorg_yolo` is more mature and advanced than the current nGraph reorg_yolo .

So we chose the 1st way to fix this issue. Some of the discussion between @XingHongChenIntel  and I can be found here: https://github.com/Flex-plaidml-team/openvino/pull/28/files (We can go the 2nd way as well.)

Reply:
Interesting, thanks for the detailed update and the link to the discussion that led to this choice. I think this is a good plan for now. I will look into what it would take to update the nGraph reference implementation of reorg_yolo and who such a change would impact.